target/linux/ath79/image/nand.mk

```diff
@@ -27,3 +27,35 @@ define Device/glinet_gl-ar300m-nor
   SUPPORTED_DEVICES += gl-ar300m glinet,gl-ar300m-nand
 endef
 TARGET_DEVICES += glinet_gl-ar300m-nor
+
+
+define Device/glinet_gl-ar750s-common
+  ATH_SOC := qca9563
+  DEVICE_VENDOR := GL.iNet
+  DEVICE_MODEL := GL-AR750S
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9887-ct \
+			kmod-usb2 kmod-usb-storage block-mount
+  KERNEL_SIZE := 2048k
+  IMAGE_SIZE := 16000k
+  PAGESIZE := 2048
+  VID_HDR_OFFSET := 2048
+endef
+
+define Device/glinet_gl-ar750s-nor-nand
+  $(Device/glinet_gl-ar750s-common)
+  DEVICE_VARIANT := NOR/NAND
+  BLOCKSIZE := 128k
+  IMAGES += factory.img
+  IMAGE/factory.img := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-ubi
+  IMAGE/sysupgrade.bin := sysupgrade-tar alt-board=gl-ar750s alt-board=glinet_gl-ar750s-nor | append-metadata
+  SUPPORTED_DEVICES += gl-ar750s glinet,gl-ar750s-nor
+endef
+TARGET_DEVICES += glinet_gl-ar750s-nor-nand
+
+define Device/glinet_gl-ar750s-nor
+  $(Device/glinet_gl-ar750s-common)
+  DEVICE_VARIANT := NOR
+  BLOCKSIZE := 64k
+  SUPPORTED_DEVICES += gl-ar750s glinet,gl-ar750s glinet,gl-ar750s-nor-nand
```

Comment:
Since ath79 is still not released yet, I would consider removing glinet,gl-ar750s here, as it is just a "work-in-progress implementation".

Reply:
It is present in v19, which is already "out in the wild", with OpenWrt-generated images having been publicly available on downloads.openwrt.org for quite some time now.

Reply:
Yes, this was only a weak suggestion. ath79 is inferior in 19.07, but yes, images are there and there are also images for master. If it was my PR, I would have removed it, but if you prefer keeping it, I'm totally fine with it and you can just mark this as resolved.